libs/mocks/build.gradle

```diff
@@ -1,21 +1 @@
-// Top-level build file where you can add configuration options common to all sub-projects/modules.
 
```

Comment:
This file ends up completely empty. Should we remove it entirely?

Reply:
Unfortunately we can't do it yet because `libs/mocks` is still a standalone project with a `settings.gradle` file. However, if/when we convert it to a simple module, we'll be able to remove it.